defi/src/adaptors/handlers/storeAdaptorData/index.ts

```diff
@@ -306,7 +306,7 @@ export const handler2 = async (event: IStoreAdaptorDataHandlerEvent) => {
 
       for (const [version, adapter] of adaptersToRun) { // the version is the key for the record (like uni v2) not the version of the adapter
         const chainBlocks = {} // WARNING: reset chain blocks for each adapter, sharing this between v1 & v2 adapters that have different end timestamps have nasty side effects
-        const runAdapterRes = await runAdapter(adapter, endTimestamp, chainBlocks, module, version, { adapterVersion })
+        const runAdapterRes = await runAdapter(adapter, endTimestamp, chainBlocks, module, version, { adapterVersion, prefetch: adaptor?.prefetch })
```

Comment:
can you instead pass the adaptor as `_module` field, this way, the we can add more field on the base level in future like allowNegativeValue and not update rest of the code